### PR TITLE
Fix zoom out

### PR DESCRIPTION
**Pull request by kailamjeter**
## Description

Fixes clear search bug (would zoom to 0,0) and adds call to fetch water system data for addresses (previously only fetched parcel data)

## Testing and Reviewing

ran locally, searched for parcel without data, cleared search

### CODE REVIEW

File: client/src/components/NationwideMap.vue [hunk 1]
```diff
@@ -345,6 +345,14 @@ export default defineComponent({
           this.marker.setLngLat([parseFloat(long), parseFloat(lat)]).addTo(this.map);
         }
         this.zoomToLongLat();
+      } else {
+        // Reset map to full view of U.S. when geo IDs are cleared.
+        if (this.map?.isStyleLoaded()) {
+          this.map?.jumpTo({
+            center: this.center,
+            zoom: 4,
```
Comment (justinebreuch) [inline client/src/components/NationwideMap.vue:353]: nit: Make zoom constant it's used twice in this file now

File: client/src/components/NationwideMap.vue [hunk 2]
```diff
@@ -323,20 +323,20 @@ export default defineComponent({
     },
     // Listens to query param to toggle layers.
     routerLayer: function(newLayer: string) {
-      if (newLayer != null) {
+      if (newLayer != null && this.map?.isStyleLoaded()) {
```
Comment (justinebreuch) [inline client/src/components/NationwideMap.vue:326]: Don't even need this watcher since app.vue dispatches an event to change the map's current layer

**Review by justinebreuch — APPROVED** [review]

File: client/src/components/NationwideMap.vue [hunk 3]
```diff
@@ -323,20 +323,20 @@ export default defineComponent({
...
         this.setDataLayerVisibility(newLayer, true);
       }
     },
     // Listen for changes to lat/long to update map location.
     'geoState.geoids': function() {
+      if (this.marker) {
+        this.marker.remove();
```
Comment (justinebreuch) [inline client/src/components/NationwideMap.vue:333]: You don't know that this marker exists yet do you?

Comment (justinebreuch) [reply client/src/components/NationwideMap.vue:333]: oh this is checking if it exists I see!

File: client/src/components/NationwideMap.vue (hunk 1 above)
Comment (kailamjeter) [reply client/src/components/NationwideMap.vue:353]: Done

File: client/src/components/NationwideMap.vue (hunk 2 above)
Comment (kailamjeter) [reply client/src/components/NationwideMap.vue:326]: Removed!

File: client/src/components/NationwideMap.vue (hunk 3 above)
Comment (kailamjeter) [reply client/src/components/NationwideMap.vue:333]: Changed to what you had anyway - it's more clear!

**Pull request merged**